.azure-pipelines/benchmarks.yml

```diff
@@ -6,7 +6,7 @@ trigger:
 
 variables:
   buildConfiguration: release
-  dotnetCoreSdkVersion: 5.0.100
+  dotnetCoreSdkVersion: 5.0.103
```

Comment:
Maybe we should use `5.0.x` to get all the minor updates

Reply:
I did that initially, and that's what the OTel version does, but in [Gitlab CI we're explicitly using 5.0.103](https://github.com/DataDog/datadog-agent-buildimages/pull/116/files#diff-dbacf0921694e46b4b06a826d1c98092f046739508ef3635251cb2b88cd57134R8-R9). Though it's probably better to keep those in sync?

Reply:
We have had issues before with using latest. A new SDJ is released that requires an update to VS, but the AzDO images are not updated, so builds fail.